python/cuml/experimental/utils.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def plot_heatmap(df_gridsearch, col1, col2):
+    max_scores = df_gridsearch.groupby([col1, col2]).max()
+    max_scores = max_scores.unstack()[['mean_test_score']]
+    sns.heatmap(max_scores.mean_test_score, annot=True, fmt='.3g')
+
+
+def plot_search_results(grid):
+    """
+    Plots by fixing all paramters to their best value
+    except the one we are plotting.
+    Params:
```

Comment:
Should follow the numpydoc docstring style here: https://www.google.com/search?q=numpydoc+style&oq=numpy&aqs=chrome.0.69i59l2j69i57j69i59j69i60l4.1438j0j7&client=ubuntu&sourceid=chrome&ie=UTF-8

See also other examples in the cuml code.

Reply:
would be nice to specify that it can take GridsearchCV or RandomizedSearchCV objects from dask.